home1:
  header:
    frame_id: "map"
  child_frame_id: "home1"
  transform:
    translation:
      x: 1.44
      y: 0.0
      z: 0.16
  euler:
    roll: 90
    pitch: 0
    yaw: 180
    
home2:
  header:
    frame_id: "map"
  child_frame_id: "home2"
  transform:
    translation:
      x: 1.235
      y: 0.0
      z: 0.16
  euler:
    roll: 90
    pitch: 0
    yaw: 180

home3:
  header:
    frame_id: "map"
  child_frame_id: "home3"
  transform:
    translation:
      x: 1.022
      y: 0.0
      z: 0.16
  euler:
    roll: 90
    pitch: 0
    yaw: 180